spec/tftp.rb

```diff
@@ -16,6 +16,11 @@
     run_test_script("host.sh")
   end
 
+  # bug 925381
+  it "checks, for unsupported YAST modules" do
```

Comment:
little note: "checks whether unsupported YAST modules are reported" (and maybe even YaST, nitpicking)